### ISSUE TITLE
resource/aws_organizations_policy: Add support for resource tags

### ISSUE THREAD

**Issue by calebplum** [terraform-providers/terraform-provider-aws#15168]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS Organizations Policies support resource tags. Adding and updating tags on a policy is supported via the AWS Organizations API with the `CreatePolicy` and `TagResource` API calls.

Terraform doesn't currently support resource tags for the `aws_organizations_policy` resource type but it would be good if it did. This would extend the degree to which these resources can be managed programmatically.

### Affected Resource(s)

- `aws_organizations_policy`

### Potential Terraform Configuration

```hcl
resource "aws_organizations_policy" "service_control_policy" {
  name        = "Sample policy"
  description = "Sample policy description"
  content     = <<CONTENT
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "RequireMicroInstanceType",
      "Effect": "Deny",
      "Action": "ec2:RunInstances",
      "Resource": "arn:aws:ec2:*:*:instance/*",
      "Condition": {
        "StringNotEquals":{               	
          "ec2:InstanceType":"t2.micro"
        }
      }
    }
  ]
}
CONTENT

  tags = {
    Environment = "Production"
    Division    = "Operations"
  }
}
```

### Expected Behavior

The AWS Organizations policy should be created or updated with the tags specified.

### Actual Behavior

Terraform returns the following `Unsupported argument` error:

```
Error: Unsupported argument

  on modules/service_control_policy/main.tf line 23, in resource "aws_organizations_policy" "service_control_policy":
   23:   tags = {

An argument named "tags" is not expected here.
```

### References

- [AWS Organizations API Reference - CreatePolicy](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreatePolicy.html)
- [AWS Organizations API Reference - TagResource](https://docs.aws.amazon.com/organizations/latest/APIReference/API_TagResource.html)

**Comment by anGie44** [comment]
`tags` support has been merged and will release with `v3.8.0` of the Provider, likely out later today.

**Comment by anGie44** [comment]
tags support has been merged and will release with v3.8.0 of the Provider, likely out later today.
